modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -516,6 +516,40 @@ export default class AvgRTPStatsReporter {
             = new AverageStatReport('stat_avg_framerate_screen_local');
 
         /**
+         * Average pixels for remote screen streaming videos (reported only if
+         * not a <tt>NaN</tt>).
+         * @type {AverageStatReport}
+         * @private
+         */
+        this._avgRemotePixels
+            = new AverageStatReport('stat_avg_pixels_remote');
+
+        /**
+         * Average pixels for remote screen streaming videos (reported only if
+         * not a <tt>NaN</tt>).
+         * @type {AverageStatReport}
+         * @private
+         */
+        this._avgRemoteScreenPixels
```

Comment:
does this refer to screenshare vs the other being camera?  looks like the comment needs to be tweaked for one of them.  also, i'd prefer naming the other one something more descriptive (`_avgRemoteCameraPixels`? and maybe this one `_avgRemoteScreensharePixels`? though this one isn't too bad as is)